src/view/com/profile/ProfileSubpageHeader.tsx

```diff
@@ -122,7 +127,14 @@ export function ProfileSubpageHeader({
             {avatarType === 'starter-pack' ? (
               <StarterPack width={58} gradient="sky" />
             ) : (
-              <UserAvatar type={avatarType} size={58} avatar={avatar} />
+              <UserAvatar
+                type={avatarType}
+                size={58}
+                avatar={
+                  avatar &&
+                  maybeModifyHighQualityImage(avatar, highQualityImages)
```

Comment:
any reason we can't apply the modification inside `UserAvatar`? It's got other usages that aren't caught in this PR, would be more surgical

Reply:
Uh... we do, actually. Oops! I'll just revert the changes on this file.